apps/api/src/domain/models/playlist/commands/add-audio-item-to-playlist/audio-item-added-to-playlist.event.ts

```diff
@@ -1,10 +1,43 @@
+import { AggregateType } from '@coscrad/api-interfaces';
 import { CoscradEvent } from '../../../../../domain/common';
+import { CoscradDataExample } from '../../../../../test-data/utilities';
+import { DTO } from '../../../../../types/DTO';
+import buildDummyUuid from '../../../__tests__/utilities/buildDummyUuid';
+import { dummyDateNow } from '../../../__tests__/utilities/dummyDateNow';
 import { BaseEvent } from '../../../shared/events/base-event.entity';
 import { AddAudioItemToPlaylist } from './add-audio-item-to-playlist.command';
 
 export type AudioItemAddedToPlaylistPayload = AddAudioItemToPlaylist;
 
+@CoscradDataExample<AudioItemAddedToPlaylist>({
+    example: {
+        // TODO why is this duplicated here? (see meta)
+        id: buildDummyUuid(1),
+        type: 'AUDIO_ITEM_ADDED_TO_PLAYLIST',
+        payload: {
+            aggregateCompositeIdentifier: {
+                type: AggregateType.playlist,
+                id: buildDummyUuid(102),
+            },
+            audioItemId: buildDummyUuid(456),
+        },
+        // Can we have a `buildTestEventMeta`?
```

Comment:
This is going to be helpful. An even better way is to use a data class for this and annotate it with an example.